packages/calypso-build/webpack.config.js

```diff
@@ -130,7 +130,7 @@ function getWebpackConfig(
 				minify: ! isDevelopment,
 			} ),
 			new DuplicatePackageCheckerPlugin(),
-			...( env.WP ? [ new WordPressExternalDependenciesPlugin() ] : [] ),
+			...( env.WP ? [ new DependencyExtractionWebpackPlugin() ] : [] ),
```

Comment:
Default behavior is probably best to include the polyfill. That's how `wp-scripts` build works.

```suggestion
			...( env.WP ? [ new DependencyExtractionWebpackPlugin( { injectPolyfill: true } ) ] : [] ),
```